lifecycle/cli.go

```diff
@@ -88,15 +99,23 @@ func create(c *cli.Context) (err error) {
 		return err
 	}
 
+	var creationSpec *spec.SpecFiles
+	if c.IsSet("spec") {
+		creationSpec, err = cliutils.GetSpec(c, true)
+		if err != nil {
+			return
+		}
+	}
```

Comment:
Asking for an input pattern via an argument (`jf rtc <release bundle name> <release bundle version> <patren>`) will likely be much simpler and provide a better user experience. I think it'll be the most common scenario.
```suggestion
	if c.IsSet("spec") {
		creationSpec, err = cliutils.GetSpec(c, true)
	} else {
		creationSpec = createDefaultReleaseBundleSpec(c) // <- implement me
	}
```

Reply:
We can consider adding this support upon requests on a later stage